README.md

```diff
@@ -3,8 +3,6 @@
 This organization is intended to host (private) source code repositories for **Babbel Hackday projects and alike**.
 It is owned and maintained by the Babbel Infrastructure Team.
 
-Please reach out to Infrastructure Team via #ask-infrastructure Channel on Slack if
+If you need access to this organizations and already are represented in [users.infrastructure](https://github.com/lessonnine/users.infrastructure), please create a PR to make yourself a member of the `babbel-hackdays` organization.
```

Comment:
I'd recommend to extend https://doc.babbel.com/lessonnine/guides.doc/tooling/github.html with `babbel-hackdays` and then link that documentation.